test/sagas/collection_test.js

```diff
@@ -101,14 +99,27 @@ describe("collection sagas", () => {
   });
 
   describe("createRecord()", () => {
-    describe("Success", () => {
-      let collection, createRecord;
+    let collection;
+
+    before(() => {
+      collection = {createRecord() {}};
+      const bucket = {collection() {return collection;}};
+      setClient({bucket() {return bucket;}});
+    });
```

Comment:
Why is this a `before` and not a `beforeEach`? When you're testing generators, I totally understand why you only want to initialize things once, but I thought it was somehow better practice to use `beforeEach` if you didn't need to initialize things once.